bedrock/firefox/templates/firefox/releases/release-notes.html

```diff
@@ -63,9 +63,9 @@ <h2>
       </a>
     {% elif release.product == 'Firefox for Android' %}
       {% if release.channel in ['Nightly', 'Aurora'] %}
-        {{ download_firefox('nightly', platform='android', alt_copy=_('Download')) }}
+        {{ download_firefox('nightly', platform='android', alt_copy=_('Download Firefox for Android Nightly')) }}
```

Comment:
Not a blocker, but if you want to make functional testing easier down the road, giving each button a unique dom id could make things simpler.